<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMeshVariableStrokeWidthWidget</class>
 <widget class="QWidget" name="QgsMeshVariableStrokeWidthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Input Values</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QgsDoubleSpinBox" name="mWidthMinimumSpinBox">
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mValueMinimumLineEdit"/>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="mValueMaximumLineEdit"/>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QPushButton" name="mDefaultMinMaxButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../images/images.qrc">
       <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Min Width</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Max Width</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Max Value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Min Value</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QgsDoubleSpinBox" name="mWidthMaximumSpinBox">
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="mIgnoreOutOfRangecheckBox">
       <property name="text">
        <string>Ignore Out of Range Values</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mUseAbsoluteValueCheckBox">
       <property name="text">
        <string>Use Absolute Value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mValueMinimumLineEdit</tabstop>
  <tabstop>mValueMaximumLineEdit</tabstop>
  <tabstop>mDefaultMinMaxButton</tabstop>
  <tabstop>mIgnoreOutOfRangecheckBox</tabstop>
  <tabstop>mUseAbsoluteValueCheckBox</tabstop>
  <tabstop>mWidthMinimumSpinBox</tabstop>
  <tabstop>mWidthMaximumSpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
